spec/coffee/directives/api/circle.spec.coffee

```diff
@@ -46,6 +46,8 @@ describe 'uiGmapCircle', ->
       visible: true
```

Comment:
This is the only file that has changes. The rest is build artifacts. I should have reverted the build; but it is too complicated to do that now.